app/components/spotlight/analytics/dashboard_component.html.erb

```diff
@@ -1,6 +1,23 @@
+<% if note %>
+  <p><%= note %></p>
+<% end %>
+
+<%= form_with url: helpers.analytics_exhibit_dashboard_path(current_exhibit), class: 'mb-3 d-flex', method: :get do |form| %>
+  <div>
+    <%= form.label :start_date, I18n.t("spotlight.dashboards.analytics.start_date_label") %>
+    <%= form.date_field :start_date, value: dates['start_date'], min: min_date, max: max_date %>
+    <%= form.label :end_date, I18n.t("spotlight.dashboards.analytics.end_date_label"), class: 'ml-2 ms-2' %>
+    <%= form.date_field :end_date, value: dates['end_date'], min: min_date, max: max_date %>
+  </div>
+  <div class="ml-auto ms-auto">
+    <%= form.submit I18n.t("spotlight.dashboards.analytics.submit_date_label"), class: 'btn btn-primary' %>
+  </div>
+<% end %>
+
 <% if results? %>
```

Comment:
Doesn't this always trigger a request to GA API even when the view itself is cached?